firmware/kivipallur_firmware_3.0/Core/Src/main.c

```diff
@@ -186,18 +186,16 @@ int32_t PIDcontrol(MotorControl* control, int16_t position){
 	return control->p_gain * error + control->i_gain * control->integraal;
 }
 
+//Throw speed control function
+uint16_t Thrower_Send(uint16_t com) {
```

Comment:
I already wrote before, but `Thrower_Send` is not changing thrower PWM and return value is also not used, when `Thrower_Send` is called.